Update index.md (#23042)
mdn/content: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md

5. Save all files and load `index.html` in the browser. Now when you click the image, it should change to the other one.

> This is what happened. You stored a reference to your {{htmlelement("img")}} element in the `myImage` identifier. Next, you made this identifier's `onclick` event handler property equal to a function with no name (an "anonymous" function). So every time this element is clicked:

Reviewer: ```suggestion
This is what happened. You stored a reference to your {{htmlelement("img")}} element in `myImage`. Next, you made its `onclick` event handler property equal to a function with no name (an "anonymous" function). So every time this element is clicked:
```
